components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -83,15 +85,29 @@ public function __construct(int $a_ref_id)
     public function executeCommand(): void
     {
         switch (strtolower($this->ctrl->getNextClass($this) ?? '')) {
+            case strtolower(ilLoginPageGUI::class):
+                $this->tabs->clearTargets();
+                $this->tabs->setBackTarget(
+                    $this->lng->txt('back'),
+                    $this->ctrl->getLinkTarget($this, 'show')
+                );
+
+                $this->context = AuthPageEditorContext::LOGIN;
+                if (strtolower($this->redirect_source ?? '') !== strtolower(ilInternalLinkGUI::class)) {
+                    $this->forwardToPageObject($this->context);
```

Comment:
Do we really need to pass this as an argument, if we already have the context stored in `$this->context`?